code/artificial_intelligence/src/linear_classifier/Least_square_classifier/Readme.md

```diff
@@ -0,0 +1,26 @@
+# Least Square Linear Classifier
+
+The idea of using least squares to create a linear classifier is to define a linear function:
+
+## Explaination
+
+We have some data that we separate into training and validation sets that we know what groups they should be in. For data in an n-dimensional space (say, the data is defined by n coordinates, or in even simpler terms, has n features), we need to find hyperplanes of dimension n-1 that will SPLIT the space in half, where everything on one side is in one group, and everything on the other side of the hyperplane is in another (Intuitively, for our purposes, you can see 2D Euclidean space can be separated into two parts by a line, which is a 1 dimensional object, and a 3D euclidean space can be split by a plane, which is a 2D object).
+
+So how do we find said hyperplane to divide up our space? The simplest way, and probably the most common way still just because it’s easy for a first-shot attempt at classification and linear regression, is to employ least-squares.
+
+## Algorithm
+
+Here the linear function taken is:
+
+f(x) = wx
```

Comment:
Since Github doesn't support math formatting, put this in indented code formatting.

```Markdown
\```
f(x) = wx
\```
```

(Don't put the slashes)